### ISSUE TITLE
Update rank-em figure functionality

### ISSUE THREAD

**Issue by jread-usgs** [USGS-VIZLAB/water-use-15#186]
once #183 is up and ready, we'll need to:
 - [x] switch the reliance on "hover" to overlap of bounding boxes when a state is dragged over a bar (hover isn't a solution that works for mobile)
 - [x] use d3.scale to set up x and y bar chart placement in the figure
 - [x] let the user know somehow that they were wrong (if they were)
 - [x] [**icing**] link the "closed" bars to their state/territory on mouseover
 - [x] [**icing**] show total value on mouseover of state or bar (when "closed")
 - [x] [**icing**] used cursor ["grab"](https://developer.mozilla.org/en-US/docs/Web/CSS/cursor) when on top of the moveable states
 - [ ] moveable states create empty `<use>` elements for locked states

**Comment by lindsaycarr** [comment]
After doing a big text overhaul, I feel like there are two things missing from this figure and would like to think about adding them if possible.

1) Text that says "Drag the states over its matching bar" that goes away as soon as you start dragging.
2) When you successfully complete the exercise, text that explains why Idaho might be a little surprising. E.g. "You might have thought that Idaho would be smaller than Oklahoma and Michigan given it's population, but it has large pockets of agriculture that increase it's total water use." (but better than that wording)

**Comment by jread-usgs** [comment]
great ideas. Similar "drag the .." text type as in climate-fish viz?

**Comment by lindsaycarr** [comment]
Yes, exactly what I was picturing.

**Comment by jread-usgs** [comment]
For font, in the climate fish viz we used 
```xml
<style type="text/css">@import url(https://fonts.googleapis.com/css?family=Shadows+Into+Light);</style
```

**Comment by lindsaycarr** [comment]
Working on some of the recent things brought up after jread's pr #217

**Comment by lindsaycarr** [comment]
Closed by #232
